app/va/va_profile/va_profile_client.py

```diff
@@ -124,6 +185,55 @@ def get_telephone(
         self.statsd_client.incr('clients.va-profile.get-telephone.failure')
         self._raise_no_contact_info_exception(self.PHONE_BIO_TYPE, va_profile_id, response.get(self.TX_AUDIT_ID))
 
+    def get_is_communication_allowed_api_v3(
+        self,
+        recipient_id: 'RecipientIdentifier',
+        communication_item_id: str,
+        notification_id: str,
+        notification_type: str,
+    ) -> bool:
+        from app.models import VA_NOTIFY_TO_VA_PROFILE_NOTIFICATION_TYPES
+
+        self.logger.info('Called get_is_communication_allowed_api_v3 for notification %s', notification_id)
+        communication_permissions: 'CommunicationPermissions' = self.get_profile(recipient_id).get(
+            'communicationPermissions', {}
+        )
+        self.logger.info(
+            f'V3 Profile -- Retrieved Communication Permissions for recipient_id: {recipient_id.id_value}, notification_id: \
+              {notification_id}, notification_type: {notification_type} -- {communication_permissions}'
+        )
```

Comment:
Same comment about not using formatted strings in logging calls.
```suggestion
        self.logger.info(
            'V3 Profile -- Retrieved Communication Permissions for recipient_id: %s, notification_id: \
              %s, notification_type: %s -- %s', recipient_id.id_value, notification_id, notification_type, communication_permissions
        )
```